I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 545, in script
    pause
  File "renpy/common/000statements.rpy", line 455, in execute_pause
    renpy.pause()
  File "game/script.rpy", line 793, in execute
    screen investigate_mode(label_name):
  File "game/script.rpy", line 793, in execute
    screen investigate_mode(label_name):
  File "game/script.rpy", line 794, in execute
    key "K_TAB" action Jump (toggle_inv_dict[label_name])
  File "game/script.rpy", line 794, in keywords
    key "K_TAB" action Jump (toggle_inv_dict[label_name])
  File "game/script.rpy", line 794, in <module>
    key "K_TAB" action Jump (toggle_inv_dict[label_name])
KeyError: '_after_load'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 545, in script
    pause
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\ast.py", line 2232, in execute
    self.call("execute")
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\ast.py", line 2220, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\statements.py", line 281, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 455, in execute_pause
    renpy.pause()
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\exports.py", line 1533, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay)
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\display\core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs) # type: ignore
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\display\core.py", line 3804, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\display\screen.py", line 451, in visit_all
    callback(self)
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\display\core.py", line 3804, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\display\screen.py", line 462, in per_interact
    self.update()
  File "C:\Users\sinha\Downloads\renpy-8.0.2-sdk\renpy\display\screen.py", line 653, in update
    self.screen.function(**self.scope)
  File "game/script.rpy", line 793, in execute
    screen investigate_mode(label_name):
  File "game/script.rpy", line 793, in execute
    screen investigate_mode(label_name):
  File "game/script.rpy", line 794, in execute
    key "K_TAB" action Jump (toggle_inv_dict[label_name])
  File "game/script.rpy", line 794, in keywords
    key "K_TAB" action Jump (toggle_inv_dict[label_name])
  File "game/script.rpy", line 794, in <module>
    key "K_TAB" action Jump (toggle_inv_dict[label_name])
KeyError: '_after_load'

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.2.22081402
DanganPineapple 1.0
Mon Sep  5 04:07:06 2022
